website/docs/r/mobile_network_packet_core_control_plane.html.markdown

```diff
@@ -0,0 +1,177 @@
+  name                = "example-mnpccp"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  sku                 = "G0"
+
+  site_ids = [azurerm_mobile_network_site.example.id]
+
+  local_diagnostics_access {
+    authentication_type = "AAD"
+  }
+
+  control_plane_access_interface {
+    name         = "default-interface"
+    ipv4_address = "192.168.1.199"
+    ipv4_gateway = "192.168.1.1"
+    ipv4_subnet  = "192.168.1.0/25"
+  }
+
+  platform {
+    type           = "AKS-HCI"
+    edge_device_id = azurerm_databox_edge_device.example.id
+  }
+
+  interop_settings = jsonencode({
+    "key" = "value"
+  })
+
+  tags = {
+    key = "value"
+  }
+
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name which should be used for this Mobile Network Packet Core Control Plane. Changing this forces a new Mobile Network Packet Core Control Plane to be created.
+
+* `resource_group_name` - (Required) Specifies the name of the Resource Group where the Mobile Network Packet Core Control Plane should exist. Changing this forces a new Mobile Network Packet Core Control Plane to be created.
+
+* `location` - (Required) Specifies the Azure Region where the Mobile Network Packet Core Control Plane should exist. Changing this forces a new Mobile Network Packet Core Control Plane to be created.
+
+* `site_ids` - (Required) A list of IDs of Mobile Network Sites in which this packet core control plane should be deployed. The Sites must be in the same location as the packet core control plane.
+
+* `sku` - (Required) The SKU defining the throughput and SIM allowances for this packet core control plane deployment. Possible values are `G0`, `G1`, `G2`, `G3`, `G4`, `G5` and `G10`.
+
+* `local_diagnostics_access` - (Required) One or more `local_diagnostics_access` block as defined below. The kubernetes ingress configuration to control access to packet core diagnostics over local APIs.
+
+* `control_plane_access_interface` - (Optional) A `control_plane_access_interface` block as defined below. The control plane interface on the access network. For 5G networks, this is the N2 interface. For 4G networks, this is the S1-MME interface.
+
+* `user_equipment_mtu_in_bytes` - (Optional) Specifies the MTU (in bytes) signaled to the UE. The same MTU is set on the user plane data links for all data networks. The MTU set on the user plane access link is calculated to be 60 bytes greater than this value to allow for GTP encapsulation.
+
+* `core_network_technology` - (Optional) The core network technology generation. Possible values are `EPG` and `5GC`.
+
+* `platform` - (Optional) A `platform` block as defined below.
+
+* `identity` - (Optional) An `identity` block as defined below.
+
+* `interop_settings` - (Optional) Settings to allow interoperability with third party components e.g. RANs and UEs.
```

Comment:
sorry missed this on my first review, as my last comment `settings ` is redundant?
```suggestion
* `interop` - (Optional) Settings to allow interoperability with third party components e.g. RANs and UEs.
```